### PR TITLE
OCLOMRS-486: Get rid of the “Remove” button for custom concepts

### PR DESCRIPTION
**Pull request by danuluma**
# JIRA TICKET NAME:
[Get rid of the “Remove” button for custom concepts](https://issues.openmrs.org/browse/OCLOMRS-486)

# Summary:
Before this, when viewing a dictionary, the "Remove" button was being shown on all the concepts. This PR eliminates this button on custom concepts.

### CODE REVIEW

File: src/components/dictionaryConcepts/components/ActionButtons.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Link } from 'react-router-dom';
 
 const ActionButtons = ({
   actionButtons,
+  source,
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/ActionButtons.jsx]: How is the above addition related to the this ticket?

File: src/components/dictionaryConcepts/components/ActionButtons.jsx [hunk 2]
```diff
@@ -15,28 +16,33 @@ const ActionButtons = ({
...
-            to={`/edit/${concept_class}/${id}${dictionaryPathName}`}
+    !retired && (
+      <React.Fragment>
+        {showExtra && (
+          <React.Fragment>
+            <Link
+              className="edit-button-link btn btn-sm mb-1 actionButtons"
+              to={`/edit/${concept_class}/${id}${dictionaryPathName}`}
+            >
+              Edit
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/ActionButtons.jsx]: How are these changes related to the getting rid of the remove button?

File: src/components/dictionaryConcepts/components/ActionButtons.jsx (hunk 1 above)
Comment (danuluma) [reply src/components/dictionaryConcepts/components/ActionButtons.jsx]: It's the source property of a concept. All the properties of a concept are being passed in props when the function is called in `src/components/dictionaryConcepts/components/ConceptTable.jsx`. Its needed in order to check if its a custom concept  or not(CIEL)

File: src/components/dictionaryConcepts/components/ActionButtons.jsx (hunk 2 above)
Comment (danuluma) [reply src/components/dictionaryConcepts/components/ActionButtons.jsx]: It was a linting fix. Wrapped the whole`React.Fragment` tag, which included the remove button, in brackets.

Comment (danuluma) [reply src/components/dictionaryConcepts/components/ActionButtons.jsx]: I removed the unnecessary changes here

**Comment by dkayiwa** [comment]
Can you take a look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by danuluma** [comment]
> Can you take a look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

I've updated the summary. If there's anything else I missed, I'd be happy to do. It will be really nice if you're specific.

**Comment by dkayiwa** [comment]
Can you ask your team mates?

**Comment by danuluma** [comment]
I've updated the commit message. I'm sorry I hadn't seen it

File: src/components/dictionaryConcepts/components/ActionButtons.jsx [hunk 3]
```diff
@@ -28,14 +28,18 @@ const ActionButtons = ({
...
       )}
-      <button
-        type="button"
-        className="btn btn-sm mb-1 actionButtons action-btn-style"
-        id="retireConcept"
-        onClick={() => { showDeleteModal(version_url); }}
-      >
-      Remove
-      </button>
+      {source !== 'Custom' && (
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/ActionButtons.jsx]: Shouldn't we have a constant for the above?

Comment (danuluma) [reply src/components/dictionaryConcepts/components/ActionButtons.jsx]: Okay let me assign it to a constant

Comment (danuluma) [reply src/components/dictionaryConcepts/components/ActionButtons.jsx]: I've assigned it to a constant `CUSTOM_SOURCE`
